content/posts/2019-03-22-creating-graphql-batch-operations-for-aws-amplify/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Creating GraphQL Batch Operations for AWS Amplify
+description: 'Use custom resolvers for bulk procedures.'
+banner: './banner.jpeg'
```

Comment:
Can you change this to:
`banner: './banner.jpg'` (instead of jpeg)

Reply:
@swaminator Done